components/ui/autocomplete/src/main/java/mozilla/components/ui/autocomplete/InlineAutocompleteEditText.kt

```diff
@@ -677,6 +677,15 @@ open class InlineAutocompleteEditText @JvmOverloads constructor(
         windowFocusChangeListener?.invoke(hasFocus)
     }
 
+    override fun onTextContextMenuItem(id: Int): Boolean {
+        var newId = id
+        // https://github.com/mozilla-mobile/fenix/issues/1833
```

Comment:
nit: Let's remote the link/comment here. I've filed a separate ticket in this repo: https://github.com/mozilla-mobile/android-components/issues/5512

Would you mind updating the commits message to reference the ticket?
Closes #5512: Strip formatting when pasting text in url bar (or similar)

Reply:
1. thx, I've updated commit message... but could you maybe put into in #5512 in the title, that it's for M and higher only? Right now there is illusion, that we cover with it everything.

2. what next?

Reply:
> what next?

:). Would be great if you could squash your commits so we can land it as one. I'll merge the change tomorrow and it will be in Fenix Nightly within 1-2 days. Nothing else needed.

> but could you maybe put info inside #5512 in the title, that it's for M and higher only
OK, I will update / leave a note on the ticket before landing.